hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/scheduler/HdfsSchedulerModule.java

```diff
@@ -0,0 +1,16 @@
+package org.apache.mesos.hdfs.scheduler;
```

Comment:
We should have the code simply call new PersistentStateStore() instead of using the guice module. This overcomplicates things since right now we have only one class inheriting the interface anyway.